client/styles/components/_about.scss

```diff
@@ -96,3 +96,9 @@
 .about__footer-list {
   padding-top: #{math.div(12, $base-font-size)}rem;
 }
+
+@media (max-width:641px){
```

Comment:
641px is not the right breakpoint.  It's 650px. 
https://github.com/processing/p5.js-web-editor/blob/91e5b79069c7d38555082b6f25bd566b8dbf74af/client/styles/components/_overlay.scss#L32